Code/Source/RobotImporter/Utils/RobotImporterUtils.h

```diff
@@ -10,22 +10,46 @@
 #include "AzCore/Component/ComponentBus.h"
 #include "AzCore/std/string/string.h"
 #include "RobotImporter/URDF/UrdfParser.h"
+#include <AzCore/IO/SystemFile.h>
 #include <AzCore/Math/Transform.h>
 #include <AzCore/std/containers/unordered_map.h>
+#include <AzCore/std/function/function_template.h>
+
 namespace ROS2
 {
+    namespace
+    {
+        const AZStd::function<bool(const AZStd::string&)> fileExistsCall = [](const AZStd::string& fn) -> bool
+        {
+            return AZ::IO::SystemFile::Exists(fn.c_str());
+        };
+
+    }
     namespace Utils
     {
+
         bool IsWheelURDFHeuristics(const urdf::LinkConstSharedPtr& link);
 
         /// Goes through URDF and finds world to entity transformation for us
-        AZ::Transform getWorldTransformURDF(const urdf::LinkSharedPtr& link, AZ::Transform t = AZ::Transform::Identity());
+        AZ::Transform GetWorldTransformURDF(const urdf::LinkSharedPtr& link, AZ::Transform t = AZ::Transform::Identity());
 
         /// Retrieve all links in urdf file
-        AZStd::unordered_map<AZStd::string, urdf::LinkSharedPtr> getAllLinks(const std::vector<urdf::LinkSharedPtr>& links);
+        AZStd::unordered_map<AZStd::string, urdf::LinkSharedPtr> GetAllLinks(const std::vector<urdf::LinkSharedPtr>& links);
 
-        /// Retrieve all links in urdf file
-        AZStd::unordered_map<AZStd::string, urdf::JointSharedPtr> getAllJoints(const std::vector<urdf::LinkSharedPtr>& links);
+        /// Retrieve all joints in urdf file
+        AZStd::unordered_map<AZStd::string, urdf::JointSharedPtr> GetAllJoints(const std::vector<urdf::LinkSharedPtr>& links);
+
+        /// Retireve all meshes as URDF paths
+        /// @param visual - find for visual
+        /// @param colliders - find for colliders
+        /// @returns set of meshes
```

Comment:
these are strings, so it is not obvious what is returned - please expand this documentation